crypto/bls12381/key.go

```diff
@@ -0,0 +1,111 @@
+//go:build !bls12381 || !((linux && amd64) || (linux && arm64) || (darwin && amd64) || (darwin && arm64) || (windows && amd64))
```

Comment:
Could we express this as the negated version of the one appearing in `crypto/bls12381/key_bls12381.go`, so that it's easier to see they are mutually exclusive?

Reply:
How?